Bottom: bd6053d1d6a647dcd0155625cf6f41dbb0d4faf4
Top:    62d47cba7b3ad41a3212be832fc70c6f8624a5ec
Author: Alexandre Oliva <aoliva@redhat.com>
Date:   2016-02-24 02:36:36 -0300

Refresh of template-support.patch

---

diff --git a/libcc1/libcp1plugin.cc b/libcc1/libcp1plugin.cc
index 1c74b94..30714ef 100644
--- a/libcc1/libcp1plugin.cc
+++ b/libcc1/libcp1plugin.cc
@@ -1039,7 +1039,7 @@ plugin_finish_record_or_union (cc1_plugin::connection *,
 				size_in_bytes) == 0);
 
   // FIXME: end_template_decl if it's a template?  I don't think so,
-  // we're only defining specializations.
+  // we're only defining specializations.  -lxo
 
   return 1;
 }
